pages/nft-book-store/new.vue

```diff
@@ -49,7 +58,7 @@ function updatePrice (e: InputEvent, key: string, index: number) {
 }
 
 function addMorePrice () {
-  prices.value.push([{ price: 0, stock: 0 }])
+  prices.value.push({ price: 0, stock: 0, name: `Tier ${prices.value.length}` })
```

Comment:
Tier starts from 0?

Reply:
expect to start at 1 because of the default value, but this is not important anyway, just a placeholder name